build/build.proj

```diff
@@ -122,6 +122,33 @@
 
   <!--
     ============================================================
+    Run VS15 apex tests
+    ============================================================
+  -->
+  <Target Name="ApexTestsStandalone"  DependsOnTargets="RestoreApex">
```

Comment:
Build usually can't run in the same msbuild process as restore, this might make it more confusing/not work.

Reply:
I think there's a way to do it now...I think runnig build with /restore does it? 
Basically it just cleans the context after restore.

Reply:
I tried this, didn't get any error. I can remove the dependsOnTarget, then we just need to restore before run